### ISSUE TITLE
训练轮数越多实际效果越差，第1、2轮效果好，以后越来越差

### ISSUE THREAD

**Issue by dengmingD** [PaddlePaddle/PaddleOCR#5510]
训练轮数越多实际效果越差，第1、2轮效果好，以后越来越差
第二次的预测结果：
[2022/02/21 11:47:09] root DEBUG: 0  Predict time of d:/888.jpg: 3.547s
[2022/02/21 11:47:09] root DEBUG: 湖南远哥食品有限公司, 0.974
[2022/02/21 11:47:09] root DEBUG: 湖南远哥食品有限公司食品有限公司湖南远哥食品有限, 0.990
[2022/02/21 11:47:09] root DEBUG: 销, 0.999
[2022/02/21 11:47:09] root DEBUG: 售, 0.998
[2022/02/21 11:47:09] root DEBUG: 合, 0.996
[2022/02/21 11:47:09] root DEBUG: 同, 0.995
[2022/02/21 11:47:09] root DEBUG: 书, 1.000
[2022/02/21 11:47:09] root DEBUG: 公司总部：湖南湘潭市雨湖区姜畲镇梅花工业园2期6栋, 0.965
[2022/02/21 11:47:09] root DEBUG: 公司总部：湖南湘潭市雨湖区姜畲镇梅花工业园2期6栋, 0.977
[2022/02/21 11:47:09] root DEBUG: 公司总部：湖南湘潭市雨湖区姜畲镇梅花工业园2期6栋, 0.979
[2022/02/21 11:47:09] root DEBUG: 招商电话：400-777-073215675277888, 0.988
[2022/02/21 11:47:09] root DEBUG: 第1页共10页, 0.997

第179次的结果：
[2022/02/21 11:36:57] root DEBUG: 0  Predict time of d:/888.jpg: 3.450s
[2022/02/21 11:36:57] root DEBUG: 督, 0.501
[2022/02/21 11:36:57] root DEBUG: 闻, 0.581
[2022/02/21 11:36:57] root DEBUG: 书, 0.949
[2022/02/21 11:36:57] root DEBUG: 公司总部：胡帝缩谭节丽溯区姜御镇梅花工业园26相, 0.749
[2022/02/21 11:36:57] root DEBUG: 公司总彰：湖南羁潭市雨湖区姜奇篡梅秘工业z期飞, 0.596
[2022/02/21 11:36:57] root DEBUG: 公司总颜：湖函潭市雨湖区桑裕篡秘工业愿2期飞, 0.635
[2022/02/21 11:36:57] root DEBUG: 暴窗电：qwvoo2.1sss2www, 0.589

第179的acc明显比第2次的差

config文件如下：
Global:
  debug: false
  use_gpu: true
  epoch_num: 1000
  log_smooth_window: 200
  print_batch_step: 100
  save_model_dir: ./output/rec_pp-OCRv2_distillation_6
  save_epoch_step: 1
  eval_batch_step: [0, 10000]
  cal_metric_during_train: true
  pretrained_model: ./train_model/ch_PP-OCRv2_rec_train/best_accuracy
  checkpoints:
  save_inference_dir:
  use_visualdl: false
  infer_img: doc/imgs_words/ch/word_1.jpg
  character_dict_path: ppocr/utils/ppocr_keys_v1.txt
  character_type: ch
  max_text_length: 15
  infer_mode: false
  use_space_char: true
  distributed: true
  save_res_path: ./output/rec/predicts_pp-OCRv2_distillation.txt

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Piecewise
    decay_epochs : [700, 800]
    values : [0.001, 0.0001]
    warmup_epoch: 5
  regularizer:
    name: L2
    factor: 2.0e-05

Architecture:
  model_type: &model_type "rec"
  name: DistillationModel
  algorithm: Distillation
  Models:
    Teacher:
      pretrained:
      freeze_params: false
      return_all_feats: true
      model_type: *model_type
      algorithm: CRNN
      Transform:
      Backbone:
        name: MobileNetV1Enhance
        scale: 0.5
      Neck:
        name: SequenceEncoder
        encoder_type: rnn
        hidden_size: 64
      Head:
        name: CTCHead
        mid_channels: 96
        fc_decay: 0.00002
    Student:
      pretrained:
      freeze_params: false
      return_all_feats: true
      model_type: *model_type
      algorithm: CRNN
      Transform:
      Backbone:
        name: MobileNetV1Enhance
        scale: 0.5
      Neck:
        name: SequenceEncoder
        encoder_type: rnn
        hidden_size: 64
      Head:
        name: CTCHead
        mid_channels: 96
        fc_decay: 0.00002

Loss:
  name: CombinedLoss
  loss_config_list:
  - DistillationCTCLoss:
      weight: 1.0
      model_name_list: ["Student", "Teacher"]
      key: head_out
  - DistillationDMLLoss:
      weight: 1.0
      act: "softmax"
      use_log: true
      model_name_pairs:
      - ["Student", "Teacher"]
      key: head_out
  - DistillationDistanceLoss:
      weight: 1.0
      mode: "l2"
      model_name_pairs:
      - ["Student", "Teacher"]
      key: backbone_out

PostProcess:
  name: DistillationCTCLabelDecode
  model_name: ["Student", "Teacher"]
  key: head_out

Metric:
  name: DistillationMetric
  base_metric_name: RecMetric
  main_indicator: acc
  key: "Student"

Train:
  dataset:
    name: SimpleDataSet
    data_dir: D:/paddle21/yu_liao_ku/train_data_6/
    label_file_list:
    - D:/paddle21/yu_liao_ku/train_data_6/train_lable.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - RecAug:
    - CTCLabelEncode:
    - RecResizeImg:
        image_shape: [3, 32, 320]
    - KeepKeys:
        keep_keys:
        - image
        - label
        - length
  loader:
    shuffle: true
    batch_size_per_card: 64
    drop_last: true
    num_sections: 1
    num_workers: 1
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: d:/ocr/test_data
    label_file_list:
    - d:/ocr/test_lable.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - CTCLabelEncode:
    - RecResizeImg:
        image_shape: [3, 32, 320]
    - KeepKeys:
        keep_keys:
        - image
        - label
        - length
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 64
    num_workers: 1

**Comment by tink2123** [comment]
是不是训练数据太少的缘故呢，看起来模型过拟合了。建议增加训练数据，或者降低学习率，减少总的迭代次数

**Comment by dengmingD** [comment]
不是数量太少，500万数量，我调一下学习率试下
